bootstrap/secret/secure.go

```diff
@@ -65,6 +66,8 @@ type SecureProvider struct {
 	securityConsulTokenDuration        gometrics.Timer
 	securityRuntimeSecretTokenDuration gometrics.Timer
 	securityGetSecretDuration          gometrics.Timer
+	httpRoundTripper                   http.RoundTripper
+	zeroTrustEnabled                   bool
```

Comment:
Since there are some functions to write those variable, the mutex should be added to protect these variable access to prevent the race condition

Reply:
see commit https://github.com/edgexfoundry/go-mod-bootstrap/pull/659/commits/d4cd1881d7d5a3bda485488f9f663197170b8a05